lib/dashboard.dart

```diff
@@ -40,6 +43,20 @@ class _DashboardState extends State<Dashboard> {
       widgets.add(const BackupSeedCard());
     }
 
+    final fakeStuff = Container(
+        padding: const EdgeInsets.only(left: 4.0, right: 4.0),
```

Comment:
🤔 I think the widgets should not be opinionated about their margins. If we need a margin I suggest we manage that on a top level, e.g. as padding on the ListView in the Scaffold. This way we don't have to maintain the margin for each widget individually, but only once.

Reply:
great idea. 👍 when the widget amount grows, things would become unwieldy.

Reply:
nit: We are talking about padding here, not margins. (padding applies to the children, margin applies to the element itself)

Note: I removed the padding on the list view. Note: The `25` padding on the overall screen that was added will have to be changed to something smaller; I use the iPhoneSE to simulate the looks and I feel `25` wastes too much space. 
There are several screens that have an overflow on iPhoneSE, I will see to add a follow up that fixes this. 
In the future we can consider making certain screen properties [responsive](https://docs.flutter.dev/development/ui/layout/adaptive-responsive) to make the most out of changing screen design :)_